openc3/lib/openc3/utilities/cli_generator.rb

```diff
@@ -116,6 +116,15 @@ def self.generate_target(args)
         false
       end
 
+      # If we're using Python create a requirements.txt and list it in the gemspec
+      File.open("requirements.txt", 'w') do |file|
+        file.puts "# Python dependencies"
+      end
```

Comment:
This will overwrite an existing requirements.txt.  Check for existence first.   Also seems to always run even if python not specified.